package.json

```diff
@@ -5,23 +5,38 @@
   "main": "index.js",
   "type": "module",
   "scripts": {
-    "build": "tsc",
-    "start": "cross-env NODE_OPTIONS=\"--loader=ts-node/esm\" node src/index.ts",
-    "dev": "nodemon",
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "build": "npx rimraf dist; tsc",
+    "start": "node dist/index.js",
+    "dev": "npm run build; nodemon",
```

Comment:
Create `storeFolder` automatically if it does not exist

![image](https://github.com/Adamant-im/ipfs-node/assets/25831507/722bbe38-ce6b-47a9-b63b-c76a94685689)